rts.tests/svscan.sh

```diff
@@ -41,7 +41,8 @@ done
 echo
 
 # stop svscan and clean up
-kill $svscanpid
+kill $svscanpid &
+wait >/dev/null 2>&1
```

Comment:
What can this possibly achieve? This runs kill in a separate process (instead of in-process on bash) and then waits for it. Does that really make something faster?

Reply:
Sorry. That shouldn't be there. It does seem to solve the problem of shell messages appearing randomly though. On Linux it gives you a hint about what happened. On BSD it just says ```Terminated```.
```
--- rts.exp     2018-09-22 05:16:40.182350033 -0400
+++ rts 2018-09-22 05:35:41.900845352 -0400
@@ -284,6 +284,7 @@
 Caught WINCH
 Caught CONT
 Caught TERM
+rts.sh: line 496:  7329 Terminated              svscan `pwd`/service > svscan.log 2>&1
 ==> svc0/output <==
 svc0 ran
```